fixtures/plugins/astro/knip.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  ignore: ['**/*.d.ts'],
```

Comment:
This `ignore` option can be removed (I need to update the docs!)

Reply:
[`05e9895` (#298)](https://github.com/webpro/knip/pull/298/commits/05e9895c49f7d373caa4122b6d846f6235188860)

I saw a changelog entry recently, just copied the config from a project of mine, sorry about that.